main.go

```diff
@@ -254,7 +254,7 @@ func main() {
 	csiMux := http.NewServeMux()
 	csiMux.HandleFunc("/healthz", healthHandler)
 	klog.Infof("Metric listening on address: /healthz")
-	if enableMetric && serviceType&utils.Node != 0 {
```

Comment:
So we only enabled metrics for node before?

Reply:
Metric for controller is disabled when I write #993, but enabled afterwards. I fail to remove this when rebasing.